metaflow/plugins/__init__.py

```diff
@@ -174,11 +174,13 @@ def get_plugin_cli():
 from .project_decorator import ProjectDecorator
 
 
+from .airflow.sensors import SUPPORTED_SENSORS
+
 FLOW_DECORATORS = [
     CondaFlowDecorator,
     ScheduleDecorator,
     ProjectDecorator,
-]
+] + SUPPORTED_SENSORS
```

Comment:
This needs rebasing on 1212. Also, there shouldn't be any `import` statement anymore (so you shouldn't have the list of supported sensors in the `.airflow.sensors` file since that would import it. You will probably have to list them individually.

Reply:
I have rebased the PR. Ready for review.